HACKING.md

```diff
@@ -39,3 +39,46 @@ To run the unit tests:
 To run the integration tests:
 
     ./integration-test.sh
+
+Importing Listenbrainz Listens to HDFS
+--------------------------------------
+
+To generate the stats and recommendations on your local machine you will need to import the listens into HDFS. It is recommended to use the ``listenbrainz-incremental-dumps`` for your local environment. Follow the given steps to import the listens.
+
+### Getting  name of the dump file
```

Comment:
in this file we're mixing -- and ## for headers, should we be consistent?

now that we're using rst for some files (for integration in sphinx?) should we consider using rst in README/HACKING too?

Reply:
I will update it to use ## instead of --. migrating these files to .rst is also a good idea. Please let me know if README.md and HACKING.md are to be migrated.

Reply:
@vansika Are these files to be migrated to .rst or should I leave them for now?

Reply:
leave them for now